components/03-modules/popup/_popup.scss

```diff
@@ -16,23 +15,25 @@ $popup__margin                   : 0 !default;
 .popup {
     display: none;
     position: fixed;
-    width: $popup__width;
-    max-width: $popup__max-width;
+    right: 0;
+    left: auto;
+    width: 100%;
```

Comment:
I think we should put this to variable `$popup__width` and for `screen-s` use other variable: `$popup__width--screen-s`. What do you think?

Reply:
Changed. Thank you, I didn't remember what we decided about variables and mq.